mm2src/coins/sia/http_client.rs

```diff
@@ -0,0 +1,208 @@
+use crate::sia::address::Address;
+use crate::sia::SiaHttpConf;
+use base64::engine::general_purpose::STANDARD as BASE64;
+use base64::Engine as _; // required for .encode() method
```

Comment:
When importing a trait, no need to alias it as _

Reply:
It's meant to limit it from being added to the namespace. Can be changed, but I prefer this style as its more easily understood IMO.